app/uploaders/article_image_uploader.rb

```diff
@@ -0,0 +1,53 @@
+class ArticleImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
+  def store_dir
+    # This will be the directory path in the CMS
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
```

Comment:
Can you change this to match our current directory structure on S3, please?

`assets/articles/YYYY/MM/DD`